src/components/menu/menu-segment-title/menu-segment-title.component.tsx

```diff
@@ -1,17 +1,25 @@
 import React, { useContext } from "react";
-import PropTypes from "prop-types";
 import StyledTitle from "./menu-segment-title.style";
 import MenuContext from "../menu.context";
 import { StyledMenuItem } from "../menu.style";
+import { VariantType } from "../menu-item";
+import tagComponent, {
+  TagProps,
+} from "../../../__internal__/utils/helpers/tags";
 
-const MenuSegmentTitle = React.forwardRef(
-  ({ children, variant = "default" }, ref) => {
+export interface MenuTitleProps extends TagProps {
+  children: string;
+  variant?: VariantType;
```

Comment:
Might be worth adding back in the comment descriptions for these props as `MenuSegmentTitle` is a consumer-facing subcomponent

Reply:
Good point thanks